src/Themer.js

```diff
@@ -146,9 +144,14 @@ export default class Themer {
    * @return {Object}           The rendered HTML with appended styles
    */
   render(snippet, props = {}) {
-    return this.resolveMiddleware(snippet)(Object.assign(props, {
-      styles: this.theme.getStyles(),
-    }));
-  }
+    const moreProps = {};
+    _.each(this.theme.getVariants(), (value, key) => {
```

Comment:
I think that variants are a base themer feature as any framework can use them - not just react.